cypress/integration/FileInput/common/index.js

```diff
@@ -0,0 +1,10 @@
+import { Given } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a single-file IputField is rendered', () => {
+    cy.visitStory('Testing:Forms', 'Standard form')
+})
+
+Given('the InputField does not hold any files', () => {
+    cy.verifyFormValue('fileTxt', undefined)
+    cy.get('.form-spy-internal')
```

Comment:
Is the .get necessary?

Reply:
Actually, no... `verifyFormValue` internally calls `getFormValue` and this already does that `get`. The get is required though to ensure the FormSpy is done rendering and the formValues have been appended to window.
I've removed this now